src/Symfony/Component/Yaml/Inline.php

```diff
@@ -52,7 +52,7 @@ public static function parse($value, $exceptionOnInvalidType = false, $objectSup
             return '';
         }
 
-        if (function_exists('mb_internal_encoding') && ((int) ini_get('mbstring.func_overload')) & 2) {
+        if (2 /* MB_OVERLOAD_STRING */ & (int) ini_get('mbstring.func_overload')) {
```

Comment:
shouldn't we check for ``extension_loaded('mbstring')`` here ? Only the extension itself can overload functions anyway